Makefile

```diff
@@ -9,43 +9,17 @@ lint:
 	cargo clippy --all -- -D warnings
 
 test:
-	@echo "------------------------------------------------------------------"
-	@echo "tests arg can be used to run specific tests"
-	@echo ""
-	@echo "for example:"
-	@echo "  make test tests=Find_references::test_returns_correct_response"
-	@echo ""
-	@echo "if no tests arg provided will run entire suite"
-	@echo "------------------------------------------------------------------"
-	@echo ""
-	@echo ""
-
-	cargo test $(tests) -- test  --nocapture
-
-manual-test: test install
-	vim me.flux
+	cargo test $(tests) -- --nocapture
 
 clean-wasm:
 	rm -rf pkg-node
 	rm -rf pkg-browser
 
-wasm: clean-wasm build
-	./build.sh
-
-wasm-local: clean-wasm
+wasm: clean-wasm
```

Comment:
Do we know why we need `clean-wasm` here? Seems kinda counterproductive to `make`, but maybe there's a reason.

Reply:
Honestly, not sure. All it does is `rm -rf` the two wasm packages we build, and I think those would get overwritten here anyway.